package.json

```diff
@@ -228,7 +229,7 @@
     "set-value": "^4.1.0",
     "source-map-support": "^0.5.19",
     "symbol-observable": "^1.2.0",
-    "tar": "^6.1.11",
+    "tar": "^6.2.1",
```

Comment:
Do we still need this now that we have `"**/geckodriver/tar": "^6.2.1"`?

Reply:
I think we should do either 1) "**/tar" or just 2) `"tar": "^6.2.1"`, depending on how tar is used. Don't need to change two places.

Reply:
Updating the  "**/tar" resolution to utilize the 6.2.1 version.